R-package/R/utils.R

```diff
@@ -59,68 +59,66 @@
 
 }
 
+# [description]
+#
+#     Besides applying checks, this function
+#
+#         1. turns feature *names* into 1-based integer positions, then
+#         2. adds an extra list element with skipped features, then
+#         3. turns 1-based integer positions into 0-based positions, and finally
+#         4. collapses the values of each list element into a string like "[0, 1]".
```

Comment:
This comment is FANTASTIC, made it very easy for me to understand the code. Thank you!